telegram/error.py

```diff
@@ -39,11 +39,14 @@ def _lstrip_str(in_s, lstr):
 class TelegramError(Exception):
     """This object represents a Telegram Error."""
 
-    def __init__(self, message):
+    def __init__(self, message, **kwargs):
```

Comment:
as you suggested, this is not a good idea :-(
see suggestion on the `ChatMigrated` code below